src/Domain/Fixture/FixtureCustomizer.php

```diff
@@ -117,4 +119,79 @@ public function removePerzParagraphsTests(FixtureOptions $options): void {
 
   }
 
+  /**
+   * Modifies DrupalKernel.php to suppress Drupal 9 deprecation warnings.
+   */
+  public function modifyDrupalKernel(FixtureOptions $options): void {
+    if (!$options->coreVersionParsedMatches('^9')) {
+      return;
+    }
+
+    $drupal_kernel_path = 'docroot/core/lib/Drupal/Core/DrupalKernel.php';
+
+    if (!$this->fixturePathHandler->exists($drupal_kernel_path)) {
+      return;
+    }
+
+    $this->output->writeln('Suppressing Drupal 9 deprecation notices for functional tests.');
+
+    $path = $this->fixturePathHandler
+      ->getPath($drupal_kernel_path);
+
+    $target = 'error_reporting(E_STRICT | E_ALL)';
+
+    $change = 'error_reporting(E_ALL & ~E_DEPRECATED)';
+
+    $this->replaceStringInFile($target, $change, $path);
+
+  }
+
+  /**
+   * Modify phpunit.xml.dist to suppress any D9 phpunit deprecation warnings.
+   *
+   * @param \Acquia\Orca\Options\FixtureOptions $options
+   *   The fixture options.
+   */
+  public function modifyPhpunitConfig(FixtureOptions $options): void {
+    if (!$options->coreVersionParsedMatches('^9')) {
+      return;
+    }
+
+    $phpunit_xml_path = 'docroot/core/phpunit.xml.dist';
+
+    if (!$this->fixturePathHandler->exists($phpunit_xml_path)) {
+      return;
+    }
+
+    $this->output->writeln('Suppressing Drupal 9 deprecation notices for phpunit tests.');
+
+    $path = $this->fixturePathHandler
+      ->getPath($phpunit_xml_path);
+
+    $target = '<ini name="error_reporting" value="32767"/>';
+
+    $change = '<ini name="error_reporting" value="24575"/>';
```

Comment:
It would be nice to have a comment explaining what this number (24575) means or why we're changing it.

Reply:
Done